modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/organization/management/v1/OrganizationManagementSuccessTest.java

```diff
@@ -773,8 +781,9 @@ public Object[][] organizationPaginationNonNumericEdgeCasesOfLimitProvider() {
             dataProvider = "organizationPaginationNonNumericEdgeCasesOfLimitDataProvider")
     public void testGetPaginatedOrganizationsForNonNumericEdgeCasesOfLimit(String limitQueryParam) {
 
-        String endpointURL = ORGANIZATION_MANAGEMENT_API_BASE_PATH + QUESTION_MARK + RECURSIVE_QUERY_PARAM + EQUAL + FALSE +
-                limitQueryParam;
+        String endpointURL =
```

Comment:
fix formatting

Reply:
Addressed with [2122b95](https://github.com/wso2/product-is/pull/21054/commits/2122b959d805379690765472e0f1c6ae56087016)